maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/dto/maat/RepOrderDTO.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.laa.crime.orchestration.dto.maat;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class RepOrderDTO {
+    private Integer id;
+    private String caseId;
+    private String catyCaseType;
+    private String appealTypeCode;
+    private String arrestSummonsNo;
+    private String userModified;
+    private LocalDate dateCreated;
+    private LocalDateTime dateModified;
+    private String magsOutcome;
+    private String magsOutcomeDate;
+    private LocalDateTime magsOutcomeDateSet;
+    private LocalDate committalDate;
+    private String decisionReasonCode;
+    private String crownRepOrderDecision;
+    private String crownRepOrderType;
+    private LocalDate crownRepOrderDate;
+    private LocalDate assessmentDateCompleted;
+    private LocalDate sentenceOrderDate;
+    private String evidenceFeeLevel;
+    private String rorsStatus;
+    @Builder.Default
+    private List<PassportAssessmentDTO> passportAssessments = new ArrayList<>();
+    @Builder.Default
+    private List<FinancialAssessmentDTO> financialAssessments = new ArrayList<>();
+    @Builder.Default
+    private List<ContributionsDTO> contributions = new ArrayList<>();
+    @Builder.Default
+    private List<RepOrderCCOutcomeDTO> repOrderCCOutcome = new ArrayList<>();
+    @Builder.Default
+    private List<IOJAppealDTO> iojAppeal = new ArrayList<>();
```

Comment:
Same here, IOJAppealDTO and ContributionsDTO are different and needs to be moved across from API. Lets create a new folder 'api' under dto/maat and move all the MAAT Court Data API's to this folder.